cli_ftd/user/heartbeat/heartbeat.c

```diff
@@ -128,13 +139,16 @@ void *heartbeat_task(void *arg0) {
     printf("LED Toggle %ld \r\n", count++);
 #endif
     GPIO_toggle(CONFIG_GPIO_GLED);
-
     body = readBpm();
+
+#if HEARTBEAT_DEBUG == 1
```

Comment:
Use the debugprintf macro instead
See stm32 module for the usage

Reply:
Ok, will address.